tests/dotnet-e2e/fileops/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿var TEXT = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.";
+TestRead();
+TestWrite();
+TestLseek();
+
+void TestRead()
+{
+    var dat = File.ReadAllText("/app/test.txt");

```

Comment:
seems like when these test are run there is a pipe created as seen in this log:
> stdout 2022-10-05T22:09:58.418189143Z 12402: 2022-10-05T22:09:58.417399Z DEBUG ThreadId(04) mirrord_layer::file::hooks: open_logic -> path "/tmp/clr-debug-pipe-12402-64281-in"
relevant issue: https://github.com/dotnet/runtime/issues/7204
could you please somehow disable this?

or else add the file pattern to this ignore regex https://github.com/metalbear-co/mirrord/blob/main/mirrord-layer/src/file.rs#L32

Reply:
Wow! Thank you for bringing this to my attention and linking the relevant issues. 

I will research this and make any necessary changes to fix this 👍🏻

Reply:
Thanks Issac, let us know if you need any help or feel free to reach out on [discord](https://discord.com/invite/J5YSrStDKD)

Reply:
I was able to resolve this with:
```
let mut env = HashMap::new();
env.insert("COMPlus_EnableDiagnostics", "0"); <--------------
env.insert("MIRRORD_AGENT_IMAGE", "test");
env.insert("MIRRORD_CHECK_VERSION", "false");
env.insert("MIRRORD_AGENT_RUST_LOG", "warn,mirrord=debug");
env.insert("MIRRORD_AGENT_COMMUNICATION_TIMEOUT", "180");
env.insert("RUST_LOG", "warn,mirrord=debug");
```

Reference:

https://github.com/dotnet/runtime/issues/7204#issuecomment-723204695

Thanks again for pointing me toward this discussion 😄

Reply:
Awesome, seems to have been fixed